src/Language/Purus/Prim/Utils.hs

```diff
@@ -69,6 +72,9 @@ polyRecordType conName vars fields =
 tyCon :: Text -> Type SourceAnn
 tyCon = TypeConstructor nullSourceAnn . primName
 
+tyConBuiltin :: Text -> Type SourceAnn
```

Comment:
Did I write this *and* builtinTyCon or did we each write one? Should probably remove one or the other.

Reply:
I'm not sure. I had two similar-looking tycon-ish things for different uses in two different places, as they needed to build different (but related) structures.